.github/workflows/api-rspec.yml

```diff
@@ -15,6 +15,23 @@ jobs:
       #   We need to test that doing this within a PR will stream that PR's
       #   log output in the test environment to the cloud logger
       STACKDRIVER_ENABLED: false
+      DATABASE_USER: storysprout_user
```

Comment:
@Donnadieu Can we set up GitHub actions secret by the same names for `DATABASE_USER` and `DATABASE_PASSWORD`? LMK when that is done so I can update the postgres service to use `${{secrets.DATABASE_USER}}` etc. 🙏🏾